stdlib/public/SDK/Foundation/Foundation.swift

```diff
@@ -1231,7 +1237,8 @@ extension NSSet {
   ///   receiver.
   @objc(_swiftInitWithSet_NSSet:)
   public convenience init(set anSet: NSSet) {
```

Comment:
Oh, I see, they have non-underscored Swift names.

These are strange initializers though.  They initialize NSArray from NSArray, NSString from NSString etc.  `@objc` names are also ugly, so maybe they could benefit from `@nonobjc`.  Anyway, this seems like a separate issue.